src/graphic/Fast3D/gfx_sdl2.cpp

```diff
@@ -414,6 +414,8 @@ static void gfx_sdl_init(const char* game_name, const char* gfx_api_name, bool s
 
 static void gfx_sdl_close(void) {
     is_running = false;
+    SDL_DestroyRenderer(renderer);
+    SDL_Quit();
```

Comment:
would it make sense to just call `gfx_sdl_destroy` here? that way when the
```
// TODO: destroy _any_ resources used by SDL
```
is addressed we wouldn't need to do things in 2 places